src/plot_profile/plot_rs/cli_rs.py

```diff
@@ -29,11 +29,12 @@
             "temp",
             "dewp_temp",
             "wind_vel",
+            "pot_temp",
         ],
         case_sensitive=False,
     ),
     multiple=True,
-    default=("temp", "dewp_temp", "wind_vel", "wind_dir"),
+    default=("temp", "dewp_temp", "wind_vel", "wind_dir", "pot_temp"),
```

Comment:
do you really want to plot pot_temp by default on every radiosounding plot?

Reply:
removed